### DISCUSSION TITLE
Could GDMP support Tflite?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [j20001970/GDMP#19] [Ideas]
[mediapipe](https://github.com/asus4/tf-lite-unity-sample/tree/master/Assets/StreamingAssets/mediapipe)

- [ ] [pose_landmark](https://github.com/asus4/tf-lite-unity-sample/blob/master/Assets/StreamingAssets/mediapipe/pose_landmark_lite.tflite)
- [ ] [hand_landmark_3d](https://github.com/asus4/tf-lite-unity-sample/blob/master/Assets/StreamingAssets/mediapipe/hand_landmark_3d_256_float16_quant.tflite)

**Comment by j20001970** [comment]
While I haven't tried these files yet, the answer should be yes.
For each MediaPipe task, we should load the **corresponding** file by specifying a .tflite (or .task) file's location or its file buffer to make the task initialize successfully.
Some .tflite files your link provides should be replaced with .task file, which is a asset bundle that contain multiple .tflite models and/or its metadata.
You can see how each task load the .tflite/.task file in [GDMP-demo](https://github.com/j20001970/GDMP-demo)
**Hand Landmarker** is already in the demo, and **Pose Landmarker** is still being worked on.

**Reply by GeorgeS2019** [reply]
>  Pose Landmarker is still being worked on.
Thank you

This project has many TFlite files. Once you have figured out one example, others could start coding for other use cases. 

Hopefully this does not require compilation from the c++ side but just the GDScript

https://github.com/asus4/tf-lite-unity-sample/tree/master/Assets/StreamingAssets/mediapipe
